docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -78,7 +78,7 @@ This will add the following to your `pom.xml`:
 
 == Writing the application
 
-Let's write a simple JAX-RS resource which has all the tokens returned in the authorization code grant response injected:
+Let's write a simple JAX-RS resource which has all the tokens returned in the authorization code grant response injected. Create `./src/main/java/org/acme/security/openid/connect/web/authentication/TokenResource.java` containing the following:
```

Comment:
I'll drop the idea of renaming the existing file, so I'd either go with Paul's suggestion or George's (with or without `src/main/java` qualification - it is fine either I'd say).
@gsmet, can you clarify what your preference is since you requested the change ?

Reply:
Well my point is that 99% of people will use an IDE. And in your IDE, you just copy paste the class and it puts it in the right package (at least that's what Eclipse does).

So I'm a bit reluctant to clutter our doc with an info that's long and in most cases not useful at all (as in an IDE, you won't use the path at all).

Reply:
I agree, I don't think this is a worthwhile change. I'll close the PR.